iati/about/templates/about/history_page.html

```diff
@@ -8,90 +8,49 @@
 <div class="hero hero--minor">
```

Comment:
Check how we are looping over the event cards, as the line seems to change depending on the text...

![image](https://user-images.githubusercontent.com/8247168/39058075-c49aeba6-44b2-11e8-8134-49e0ce109fa1.png)

![image](https://user-images.githubusercontent.com/8247168/39058120-de4cbdb8-44b2-11e8-9e50-9e0bcb39a5ae.png)

Reply:
Related issue https://github.com/IATI/IATI-Pattern-Library/issues/23

Reply:
@dalepotter This is an issue with the design that fffunction have now addressed: https://github.com/IATI/IATI-Pattern-Library/issues/23 the fix can be applied outside of this branch